tutorials/javaInstallationMac.md

```diff
@@ -0,0 +1,79 @@
+{% set title="Java 17 Installation Guide for Mac Users" %}
+<frontmatter>
+  title: "{{ title }}"
+  pageNav: 2
+</frontmatter>
+
+# {{ title }}
+
+## Introduction
+
+This guide provides detailed steps to install Java JDK-FX 17.0.11.fx-zulu on macOS using SDKMAN.
+
+## Installation Steps
+
+### Step 1: Install SDKMAN
+
+1. Open Terminal and install SDKMAN by running:
+   ```bash
+   curl -s "https://get.sdkman.io" | bash
+   ```
+2. Follow the instructions on the screen to complete the installation.
+3. Restart your terminal or run:
+   ```bash
+   source "$HOME/.sdkman/bin/sdkman-init.sh"
+   ```
+
+### Step 2: Install JDK-FX Using SDKMAN
+
+1. Install the required version (e.g., 17.0.11.fx-zulu) by running:
+   ```bash
+   sdk install java 17.0.11.fx-zulu
+   ```
+2. This version should be automatically set as the default, but if it is not, set it as the default by running:
+   ```bash
+   sdk default java 17.0.11.fx-zulu
+   ```
+
+### Step 3: Verify Installation
+
+1. Verify the installation by running:
+   ```bash
+   java -version
+   ```
+   You should see the version information for Java 17.0.11.fx-zulu.
+
+## Switching Between Java Versions
+
+If you have multiple versions of Java installed, you can switch between them using SDKMAN.
+
+You can list the currently active Java version by running:
+   ```bash
+   sdk current java
+   ```
+
+### Switch to a Different Java Version
+
+1. To switch to a different installed version, use:
```

Comment:
This might be clearer:
1. You can switch to any installed java version using:
   ```bash
   sdk use java <version>
   ```
   To see all available versions, type `sdk use java` then press the `TAB` key. Keep pressing `TAB` until you select the desired version.

Reply:
this would be clearer